src/Sfa.Tl.Matching.Database/Tables/Opportunity.sql

```diff
@@ -7,6 +7,7 @@
 	[JobTitle] NVARCHAR(250) NULL,
 	[PlacementsKnown] BIT NULL,
 	[Placements] INT NULL,
+	[EmployerId] INT NULL,
```

Comment:
Is this going to be OK if the Employer table is wiped? Would it be better to use the EmployerCrm?

Reply:
either employerid or crmId would cause issue if the table is wiped in the middle of the journey
it wont be a problem if that table is wiped once the journey is completed

Reply:
I didn't think EmployerCrm was a value that was changed? So a wipe and recreate would produce the same value?

True...as long as the journey is complete, then it won't matter.